templates/adherent/home.html.twig

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title 'Bienvenue, ' ~ app.user.firstName %}
+
+{% block content %}
+    <main>
+        <section class="content b__nudge--bottom-huge">
+            <header class="l__wrapper--slim space--60-0">
+                <h1 class="text--center text--large">
+                    Bienvenue, {{ app.user.firstName }} ! 👋
```

Comment:
I don't know anything about this. Is it ok to use unicode emoji directly in the HTML ? Are they going to be rendered correctly in most cases ?

Reply:
Yeah. It is, plus it's old code.

Reply:
Yes it's unicode.

Reply:
thanks ;-)